RPG Wizardry/Assets/Scripts/Player/Inventory/PlayerInventory.cs

```diff
@@ -104,6 +102,24 @@ public void RemoveGoldListener(OnInventoryChange listener)
         {
             goldChangeEvent -= listener;
         }
+
+        /// <summary>
+        /// Add Listener from Page-Event
+        /// </summary>
+        /// <param name="listener">Listener to Add</param>
+        public void AddPageListener(OnInventoryChange listener)
```

Comment:
You probably want a seperate delegate here, so you can pass the Page to the listeners.